src/components/UserProfileCard.jsx

```diff
@@ -0,0 +1,25 @@
+import profileImage from '@assets/icon/ic_user.png';
+import styles from '../styles/UserProfilCard.module.css';
+
+const classNameByauthority = {
+  post: styles.post,
+  comment: styles.comment,
+};
+
+function UserProfileCard({ authority, name, time, image }) {
+  const userImage = image ? image : profileImage;
```

Comment:
```suggestion
  const userImage =  image || profileImage;
```

이렇게 좀 더 단순하게 쓸 수 있겠네요!